.pre-commit-config.yaml

```diff
@@ -77,27 +77,6 @@ repos:
             --linelength=88,
             '--filter=-readability/casting,-runtime/int,-build/include_subdir,-readability/fn_size'
         ]
--   repo: https://github.com/pycqa/pylint
```

Comment:
Why did this change?

Reply:
Ups, I've added it back in the newest version.